desktop/app/env.js

```diff
@@ -1,6 +1,4 @@
-/**
- * External Dependencies
- */
+/* eslint-disable import/order */
```

Comment:
It seems like the order of `require`s might be relevant which is why I disabled the rule for this file.